hr_timesheet_overtime/models/hr_employee.py

```diff
@@ -92,14 +98,14 @@ def _compute_has_overtime_access(self):
                 "hr.group_hr_manager"
             ) or self.env.user.has_group("hr.group_hr_user"):
                 has_access = True
```

Comment:
you can use `self._has_overtime_write_access()` here too.

Reply:
fixed